sharktank/sharktank/models/llama/llama.py

```diff
@@ -132,25 +132,6 @@ def prefill(
         self._assert_device(seq_block_ids)
         self._assert_device(*cache_state, dtype=self.activation_dtype)
 
-        if self.config.tensor_parallelism_size > 1:
```

Comment:
It would be nice to leave asserts for the expected argument types.

Reply:
We want to avoid bleeding too much sharding code into the modelling work. Too much mixing make it difficult for straight modelling people to work through.